lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -44,6 +52,39 @@ def setUp(self):
 
         self.namespaced_url = 'certificates_api:v0:certificates:detail'
 
+        # create a configuration for django-oauth-toolkit (DOT)
+        dot_app_user = UserFactory.create(password=USER_PASSWORD)
+        self.dot_app = dot_models.Application.objects.create(
+            name='test app',
+            user=dot_app_user,
+            client_type='confidential',
+            authorization_grant_type='authorization-code',
+            redirect_uris='http://localhost:8079/complete/edxorg/'
+        )
+        self.dot_access_token = dot_models.AccessToken.objects.create(
+            user=self.student,
+            application=self.dot_app,
+            expires=datetime.utcnow() + timedelta(weeks=1),
+            scope='read write',
+            token=self.get_token(30)
+        )
+        self.dot_refresh_token = dot_models.RefreshToken.objects.create(
+            user=self.student,
+            application=self.dot_app,
+            access_token=self.dot_access_token,
+            token=self.get_token(30)
+        )
+
+    def get_token(self, str_len):
```

Comment:
Actually, there isn't much value to having this here. Just make a token up in your calls to `create()`.